dll/win32/oledlg/lang/oledlg_Zh.rc

```diff
@@ -107,52 +109,52 @@ LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_TRADITIONAL
 
 STRINGTABLE
 {
-  IDS_RESULTOBJDESC	"在你的檔案中插入一個新的 %s 對象"
-  IDS_RESULTFILEOBJDESC	"將檔案的內容以對象的方式插入到你的檔案以便你可以用創建本檔案的程式來激活它."
+  IDS_RESULTOBJDESC	"在你的檔案中插入一個新的 %s 物件"
+  IDS_RESULTFILEOBJDESC	"將檔案的內容以物件的方式插入到您的文件以便您可以用程式來啟動它。"
   IDS_BROWSE            "瀏覽"
-  IDS_NOTOLEMOD         "本檔案不像是一個有效的 OLE 模型. 不能註冊 OLE 控件."
-  IDS_NOTOLEMODCAPTION  "添加控件"
-  IDS_VERBMENU_OBJECT   "對象 (&O)"
```

Comment:
is this white space unnecessary？

Reply:
since these lines are "strings to be removed", so I guess you mean the new string `IDS_VERBMENU_OBJECT   "物件(&O)"` .🤔

If so, my answer is: **yes, not necessary**